published/005-2016-Feb-1.md

```diff
@@ -0,0 +1,105 @@
+# IPFS Weekly #5
+
+[IPFS](//ipfs.io/) is a new hypermedia distribution protocol, addressed by content and identities, aiming to make the web faster, safer, and more open. In these posts, we highlight some of the development that has happened in the past week. For anyone looking to get involved, follow the embedded hyperlinks, search the wealth of information on [GitHub](//github.com/ipfs) or join us on [IRC](//webchat.freenode.net/?channels=ipfs) (#ipfs on the Freenode network).
+
+If you would like to get this update as an email, sign up for our [weekly newsletter](//tinyletter.com/ipfsweekly)!
+
+Here are some of the highlights for the [February 1](//github.com/ipfs/pm/issues/88) sprint:
+
+## Updates
+
+* [**go-ipfs**](//github.com/ipfs/go-ipfs) 
+	- [@noffle](//github.com/noffle) made [a fix](https://github.com/ipfs/go-ipfs/pull/2296) for when your FUSE connection is terminated
+	- [@lgierth](//github.com/lgierth) automated the [docker image](https://github.com/ipfs/go-ipfs/pull/2308) for go-ipfs 
+	- [@RichardLitt](//github.com/RichardLitt) made a lot of documentation fixes to the CLI while working on the IPFS HTTP API.
+* [**distributions**](//github.com/ipfs/distributions)
+	- [dist.ipfs.io](http://dist.ipfs.io/) now has [signed releases](https://github.com/ipfs/distributions/pull/51) thanks to @dignifiedquire
+	- [@noffle](//github.com/noffle) [worked](https://github.com/ipfs/distributions/pull/42) to get [ipget](http://ipget.me/) ready to be on [dist.ipfs.io](http://dist.ipfs.io/)
+* [**gx**](//github.com/whyrusleeping) 
+	- [@whyrusleeping](//github.com/whyrusleeping) shipped the GX PR, which pulls relevant code out of go-ipfs and into the new GX package manager. He also wrote a tool to publish IPNS records without the IPFS daemon.
```

Comment:
That PR also includes libp2p doesn't it? Wanna write a few words @whyrusleeping? :)